core/templates/pages/exploration-editor-page/translation-tab/services/translation-status.service.ts

```diff
@@ -147,16 +148,19 @@ angular.module('oppia').factory('TranslationStatusService', [
               if (TranslationTabActiveModeService.isTranslationModeActive()) {
                 stateNeedsUpdateWarnings[stateName] = (
                   TRANSLATION_NEEDS_UPDATE_MESSAGE);
+                stateNeedsUpdate = true;
               } else {
                 stateNeedsUpdateWarnings[stateName] = (
                   AUDIO_NEEDS_UPDATE_MESSAGE);
+                stateNeedsUpdate = true;
               }
             }
           });
           explorationContentNotAvailableCount += noTranslationCount;
-          if (noTranslationCount === 0) {
+          if (noTranslationCount === 0 && !stateNeedsUpdate) {
```

Comment:
```if(!noTranslationCount && !stateNeedsUpdate) {``` ?

Reply:
Thanks @Showtim3! I'll address these comments in my next PR since these comments are nits and this PR is required for the release cut in a few hours.